src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStartedExtensionPoint.ts

```diff
@@ -27,6 +27,10 @@ export const walkthroughsExtensionPoint = ExtensionsRegistry.registerExtensionPo
 					type: 'string',
 					description: localize('walkthroughs.title', "Title of walkthrough.")
 				},
+				icon: {
```

Comment:
Will this render theme icons e.g. `$(iconName)` like the built-in walkthroughs do?

Reply:
Doesn't look like this PR currently supports that. Is this something you think we should add support for? Right now we allow them in various places within the walkthrough, but only built-ins can have them in the actual icon slot